CMakeLists.txt

```diff
@@ -1,16 +1,23 @@
-cmake_minimum_required(VERSION 3.23)
-project(farm_ng_core VERSION 22.99.0)
-
-list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+cmake_minimum_required(VERSION 3.16)
+# https://semver.org/
+project(farm_ng_core VERSION 0.0.1)
```

Comment:
If we follow semver, we should start with version 0.1.0

https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

Reply:
Thanks fixed.